spec/sidekiq/form526_confirmation_email_job_spec.rb

```diff
@@ -70,6 +71,7 @@
           }
         }
         allow(Notifications::Client).to receive(:new).and_return(notification_client)
+        allow(VaNotify::Client).to receive(:new).and_return(va_notify_client)
```

Comment:
Can this be DRYed up by moving `allow(VaNotify::Client).to receive(:new).and_return(va_notify_client)` to the `context`block